Controllers/HomeController.cs

```diff
@@ -71,6 +75,7 @@ public IActionResult Result(string status, [FromQuery(Name = "reason")] string r
         }
 
         [ResponseCache(Duration = 0, Location = ResponseCacheLocation.None, NoStore = true)]
+        [Route("error")]
```

Comment:
I think we use /result/error in other samples ? https://github.com/adyen-examples/adyen-node-online-payments/blob/main/index.js

Reply:
This is the default error page. The /result/error page is part of the [HttpGet("/result/{status}")] function, which is resolved properly now 👍